lib/scope.js

```diff
@@ -144,7 +145,7 @@ ScopeDefinition.prototype.related = function(receiver, scopeParams, condOrRefres
           queryRelated.where[IdKey] = collectTargetIds(data, IdKey);
         }
 
-        relatedModel.find(queryRelated, cb);
+        relatedModel.find(queryRelated, options, cb);
```

Comment:
I agree that `options` should be passed to both `relatedModel.find` and `targetModel.find` methods, because we are using `options` to propagate context data like current access token, current (SQL) transaction, etc.